hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/MetricsRegionWrapperImpl.java

```diff
@@ -313,9 +341,24 @@ public void run() {
             tempVal += store.getMemstoreOnlyRowReadsCount();
           }
           readsOnlyFromMemstore.put(store.getColumnFamilyName(), tempVal);
+
+          Map<String, Pair<Long, Long>> sfAccessTimeAndSizeMap =
+            store.getStoreFilesAccessTimeAndSize();
+          Long now = EnvironmentEdgeManager.currentTime();
```

Comment:
Use `long` instead of `Long` as later we will do calculation?

Reply:
> Use `long` instead of `Long` as later we will do calculation?

OK, sir.

Reply:
OK, sir.